cip-0016.md

```diff
@@ -0,0 +1,127 @@
+<pre>
+CIP: 16
+Title: Scheduled Distributions
+Author: Dan Anderson <me@dananderson.org>
+Discussions-To: https://counterpartytalk.org/t/cip-proposal-scheduled-distributions/4449
+Status: Draft
+Type: Standards Track
+Created: 2018-03-15
+</pre>
+
+## Abstract
+Allow dividends to be scheduled to activate on locked assets, by defining future block height and escrowing funds.
+
+## Specification
+Mirroring the approach used by orders and order expirations, this CIP extends dividends to include an optional activation index. As part of this improvement, a dividend_activations table will be utilized, as well as, a new dividend status equal to 'pending.' These three changes, combined with new validation rules and message encoding, constitute this feature.
+
+### Effects of CIP
+After this change, when an activation index is not provided, dividends will be distributed, as they have been, immediately. However, when an activation index is provided, the quantity needed to fund that distribution will become escrowed and those funds will not be distributed until the current block height equals the given activation index.
+
+**Important to note:** Unlike orders, which can be canceled before their expiration index, scheduled dividends cannot be canceled. **Also important to note:** Recipients of the scheduled dividend will be calculated at the time of activation, not before.
```

Comment:
There is an unbounded issue here: if the amount of holders * dividend amount is higher than the current holdings, what happens with the dividend? Would it make sense to determine the TOTAL distribution and then just subdivide the distribution?